posthog/async_migrations/migrations/0004_replicated_schema.py

```diff
@@ -87,12 +87,9 @@ class Migration(AsyncMigrationDefinition):
     depends_on = "0003_fill_person_distinct_id2"
 
     def is_required(self):
-        return "Distributed" not in self.get_current_engine("events")
+        return "Distributed" not in cast(str, self.get_current_engine("events"))
 
     def precheck(self):
```

Comment:
Add to the pre-check a check about backup table existing similar to https://github.com/PostHog/posthog/pull/8556

Reply:
actually ignore this you are renaming it back to the tmp name which is different from what we did in events to keep the failed attempt around